server/lib/config/pbench-server-default.cfg

```diff
@@ -122,6 +122,12 @@ webserver = %(default-host)s
 host-info-path-prefix = pbench-results-host-info.versioned/pbench-results-host-info.URL
 host_info_url = http://%(webserver)s/%(host-info-path-prefix)s
 
+# hostport logger type uses UDP-based logging.
+[logging]
+logger_type = file
```

Comment:
Can we make the default `devlog`?  And then fix all the other unit tests to use `file` as necessary?

Reply:
Changing the default to devlog doesn't make any difference to unit-tests as they have their own config files.